docs/localization/LOCALIZATION.md

```diff
@@ -29,7 +29,7 @@ Localization module is built on top of an open source [ngx-translate](https://gi
 
 # How to use
 
-We provide in [library](https://github.com/AmadeusITGroup/otter/blob/main/packages/@o3r/localization/src/tools/localization.module.ts) an angular module called **LocalizationModule** which comes with translations loader.
+We provide in [library](https://github.com/AmadeusITGroup/otter/blob/main/packages/@o3r/localization/src/tools/localization.module.ts) an Angular module called **LocalizationModule** which comes with translations loader.
 
 - **In your AppModule** you need to **import** the **LocalizationModule** and **TranslateModule**. The LocalizationModule could be imported calling `forRoot` with a custom configuration **factory** to specify the language of the application. This configuration is of type **LocalizationConfiguration** and describes your endpoint URL, supported locales, list of RTL languages, the language of your application and your fallback language.
```

Comment:
```suggestion
- **In your AppModule** you need to **import** the ``LocalizationModule`` and ``TranslateModule``. The LocalizationModule could be imported calling `forRoot` with a custom configuration **factory** to specify the language of the application. This configuration is of type ``LocalizationConfiguration`` and describes your endpoint URL, supported locales, list of RTL languages, the language of your application and your fallback language.
```

Reply:
Method names and functions should be between `` shouldn't they?
(Comment for other parts of the file)